src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: all ease 3s;
```

Comment:
It's not recommended to use 'all' in the transition property. It can cause performance issues. Specify which properties you want to animate. Also, 3s seems too long for a transition, consider reducing it.